napari/layers/surface/surface.py

```diff
@@ -448,3 +452,68 @@ def _get_value(self, position):
             Value of the data at the coord.
         """
         return None
+
+    def _get_value_3d(
+        self,
+        start_point: np.ndarray,
+        end_point: np.ndarray,
+        dims_displayed: List[int],
+    ) -> Tuple[Union[None, float, int], None]:
+        """Get the layer data value along a ray
+
+        Parameters
+        ----------
+        start_point : np.ndarray
+            The start position of the ray used to interrogate the data.
+        end_point : np.ndarray
+            The end position of the ray used to interrogate the data.
+        dims_displayed : List[int]
+            The indices of the dimensions currently displayed in the Viewer.
+
+        Returns
+        -------
+        value
+            The data value along the supplied ray.
+        vertex : None
+            Index of vertex if any that is at the coordinates. Always returns `None`.
+        """
+        if len(dims_displayed) != 3:
+            # only applies to 3D
+            return None, None
+        if (start_point is None) or (end_point is None):
+            # return None if the ray doesn't intersect the data bounding box
+            return None, None
+
+        start_position, ray_direction = nd_line_segment_to_displayed_data_ray(
+            start_point=start_point,
+            end_point=end_point,
+            dims_displayed=dims_displayed,
+        )
+
+        # get the mesh triangles
+        mesh_triangles = self._data_view[self._view_faces]
+
+        # get the triangles intersection
+        intersection_index, intersection = find_nearest_triangle_intersection(
+            ray_position=start_position,
+            ray_direction=ray_direction,
+            triangles=mesh_triangles,
+        )
+
+        if (intersection_index is None) or (intersection is None):
```

Comment:
The docstring seems to imply this will always be `None` as this is the second return value. If that's always true, consider skipping this check and maybe even using the standard anonymous variable `_` to avoid ever defining the variable? If it's not, then fix the docstring?

Reply:
Good point. `intersection_index` and `intersection` should always be both a value or both None. I've applied the fix you suggested.